v1.0/openapi3/schemas/AuthenticationType.yaml

```diff
@@ -3,4 +3,6 @@ type: string
 enum:
   - OTP
   - QRCODE
+  - U2F
```

Comment:
Woah now. Mojaloop API v1.0 is set in stone. We can't add a new enumerator like that.
API v1.1 is supposed to have the U2F change (I believe).
We, as the pisp stream will have to define out own `AuthenticationType` for now in pisp api specifications.

I've been thinking of a pisp-flavoured folder in the interim but please undo any major changes to in this PR.

Reply:
Thanks Kevin.

Yes, it will happen in v2.0.. (v1.1 was published in May 2020)

Reply:
Ah awesome. I should start adding v1.1 then.

Reply:
Ohh okay, sorry didn't intend to include this change in the PR and just realized that it is there...  Will update this PR soon

Reply:
Reverted this update